bin/pycbc_inspiral

```diff
@@ -25,6 +25,7 @@ from pycbc.types import TimeSeries, FrequencySeries, zeros, float32, complex64
 import pycbc.fft.fftw, pycbc.version
 import pycbc.opt
 import pycbc.weave
+import pycbc.injcutter
```

Comment:
I think I'm seeing Duncan's point on the name now too. It seem backwards and thats a bit confusing. It is actually rejecting templates, not cutting injections. I suggest TemplateFilterRejector. @duncan-brown Any better idea?